SWLOR.Game.Server/Feature/SpawnDefinition/ViscaraSpawnDefinition.cs

```diff
@@ -224,5 +225,14 @@ private void VelesSewers()
                 .ReturnsHome();
         }
 
+        private void TwilightPraxeum()
+        {
+            _builder.Create("REV_MAZE")
```

Comment:
Please follow the established naming scheme for spawns. Look at other spawns in this file to get an idea. It should lead with the planet, followed by a short descriptor of where it's used. 

I.E: VISCARA_REVANITE_MAZE  would suffice.

Reply:
Changed.